subsys/task_wdt/task_wdt_syswq_stall.c

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/task_wdt/task_wdt.h>
+
+LOG_MODULE_REGISTER(task_wdt_syswq_stall, CONFIG_LOG_DEFAULT_LEVEL);
+
+static const uint32_t timeout_ms = CONFIG_TASK_WDT_SYSWQ_STALL_TIMEOUT_MS;
+static const uint32_t feed_delay_ms = CONFIG_TASK_WDT_SYSWQ_STALL_TIMEOUT_MS / 2;
```

Comment:
No need for these additional variables, simply use the Kconfig symbols directly.